packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -292,9 +292,9 @@ export function FileSnapshotWriterClassFactory<TBase extends ReaderConstructor>(
             }
 
             for (const blobName of Object.keys(snapshotTree.blobs)) {
-                const contents = await this.read(snapshotTree.blobs[blobName]);
+                const contents = await this.readString(snapshotTree.blobs[blobName]);
```

Comment:
FYI - This code does not look right (before your changes) for the similar reason to code in prefetcher being wrong, as it incorrectly assumes that all blobs are strings. I think we leave it as is, and whenever we get files with binary blobs, we will need to fix it.